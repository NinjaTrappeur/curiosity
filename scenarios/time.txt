reset
time
time --step
time --step --minute
time
