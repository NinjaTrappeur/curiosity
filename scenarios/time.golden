1: reset
State is now empty.
2: time
01/Jan/1970:01:00:00 +0100
3: time --step
01/Jan/1970:01:00:01 +0100
4: time --step --minute
01/Jan/1970:01:01:00 +0100
5: time
01/Jan/1970:01:01:00 +0100
